integration_tests/src/main/python/conftest.py

```diff
@@ -113,7 +113,6 @@ def is_parquet_testing_tests_forced():
 _inject_oom = None
 
 def should_inject_oom():
-    global _inject_oom
```

Comment:
This is something I owed @gerashegalov from a long time ago https://github.com/NVIDIA/spark-rapids/pull/7925#discussion_r1153257428.

Since we don't need to change the variable, it's not needed.